daemon/src/setup_contract.rs

```diff
@@ -218,39 +388,42 @@ pub async fn new(
         &commit_tx,
         &lock_desc,
         lock_amount,
-        &msg1.commit,
+        commit_sig,
         &params.own_punish.publish_pk,
         &params.counterparty.identity_pk,
     )
     .context("Commit adaptor signature does not verify")?;
 
-    for own_grouped_cets in own_cets.clone() {
+    {
```

Comment:
unneeded set of braces?

Reply:
This is to make sure that `verify_own` is dropped after the for loop finishes its last iteration

Reply:
maybe we could add a comment to that extent, so it's not that confusing?